beacon/validation/fields.py

```diff
@@ -219,7 +219,10 @@ def __init__(self, *, items=None, separator=',', trim=True, **kwargs):
     async def convert(self, value: str, **kwargs) -> set:
         if value in EMPTY_VALUES:
             return self.default
-        values = value.split(self.separator)
+        try:
```

Comment:
Do not use a blind try-except block, you would catch any exception in here and make it hard to understand what is happening. You can test the type of the variable to do one thing or another instead.